web/lib/context/Settings/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { createContext, useState, useEffect } from 'react'
+import { getExampleSpeechSettings } from '../../storage'
+
+export const SettingsContext = createContext()
+
+export const SettingsProvider = ({ children }) => {
+  const [state, setState] = useState({
+    showExampleSpeech: false,
+  })
```

Comment:
Let's define `[showSpeechForExamples, setShowSpeechForExamples]` instead of a `state` object. It's faster that way.